### PR TITLE
dev-libs/device-atlas-api-c: version bump to 2.1.2_2

### PR DESCRIPTION
**Pull request by devnexen**

### CODE REVIEW

**Comment by a17r** [comment]
There's two possibilities here:
a) Everyone is scrambling to get a look at your change for curiousness
b) No one is looking at this because summary is meaningless.

Please fix your git summary and PR title. :)

**Comment by devnexen** [comment]
Hi, was wondering since @bjacquin seems inactive since months, could I maintain this package since I develop this library anyway ? I maintain also few openbsd packages I have little experience. Cheers.

**Comment by bjacquin** [comment]
@devnexen Thank you for your interest, but I am not a Gentoo developers ;) I'm not sure I have much value here

**Comment by devnexen** [comment]
@bjacquin No worries I understand if possible I can take maintainer ship to relieve if possible.

File: dev-libs/device-atlas-api-c/device-atlas-api-c-2.1.2_p2.ebuild [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+MY_P="deviceatlas-enterprise-c-${PV/_p/_}"
+
+DESCRIPTION="API to detect devices based on the User-Agent HTTP header"
+HOMEPAGE="https://deviceatlas.com"
+SRC_URI="${MY_P}.tgz"
+
+LICENSE="all-rights-reserved"
+SLOT="0"
+KEYWORDS="amd64 ~arm ppc x86"
```
Comment (monsieurp) [inline dev-libs/device-atlas-api-c/device-atlas-api-c-2.1.2_p2.ebuild]: This is not correct. Please drop those stable versions next time. I will fix this during the merge.

**Pull request closed without merging**